core/pom.xml

```diff
@@ -35,7 +35,7 @@
         <javac.target>11</javac.target>
         <argLine>-ea -Dfile.encoding=UTF-8</argLine>
         <test.exclude>None</test.exclude>
-        <web.console.version>0.0.3</web.console.version>
+        <web.console.version>0.0.5</web.console.version>
```

Comment:
why did we skip a version?

Reply:
because we forgot about 0.0.4

Reply:
i mean, 0.0.4 was released, but not added to this repo

Reply:
ok, cool